src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -0,0 +1,74 @@
+
+Below is a list of event-level keys and tokens reserved and known to Sentry:
+
+| Key/Token                     | Description |
+|---------------------------| ----------- |
+| location | Restrict results to the events with a matching location. |
+| message | Restrict results to events with a matching message. |
+| environment | Restrict results to events tagged with the matching environment. |
+| release | Restrict results to events tagged with the matching release. You can create a token with an exact match on the version of a release, or `release:latest` to pick the most recent release. |
+| transaction | Restrict results to events tagged with a URL template/job name. |
+| geo.country_code<br /> geo.region geo.city | Restrict results to events triggered by a geographic area. |
+| http.method<br /> http.referer<br /> http.url | Restrict results based on the HTTP request context. |
+| user.id user.email<br /> user.username<br /> user.ip | Restrict results to events affecting a given user. |
+| event.timestamp | Restrict results to events that occurred at the given timestamp. This filter can be passed twice to provide a range.<br /><br /> Events occurred on January 2, 2016: `event.timestamp:2016-01-02`<br /><br /> Events between 01:00 and 02:00 (UTC): `event.timestamp:>=2016-01-02T01:00:00 event.timestamp:<2016-01-02T02:00:00`<br /><br /> The following comparative operators are available: greater than (`>`), greater than or equal (`>=`), less than (`<`), less than or equal (`<=`) |
+| device.arch<br /> device.battery_level<br /> device.brand<br /> device.charging<br /> device.locale<br /> device.model_id<br /> device.name<br /> device.online<br /> device.orientation<br /> device.simulator<br /> device.uuid | Restrict results to events tagged with a specific device attribute. |
+| os.build<br /> os.kernel_version | Restrict results to events tagged with a specific operating system property. |
+| stack.abs_path<br /> stack.filename<br /> stack.function<br /> stack.module<br /> stack.package<br /> stack.stack_level<br /> stack.lineno | Restrict results to events with a matching stack property.<br /><br /> For Native SDK users, `stack.package` is the Native equivalent for `stack.module`. For more details about enhancing search, see the full documentation on [adding context and customizing tags](/platforms/native/enriching-events/). |
+| error.type<br /> error.value<br /> error.mechanism<br /> error.handled | Restrict results to events with a matching error property. |
+
+### Custom Tags
+
+Additionally, you can use any tag you’ve specified as a token. Tags are various key/value pairs that get assigned to an event, and you can use them later as a breakdown or quick access to finding related events.
+
+Most SDKs generally support configuring tags by configuring the scope:
+
+<PlatformContent includePath="sensitive-data/set-tag" />
```

Comment:
This include doesn't work properly :( 

Can you instead link to the platform-specific content for tags? For example, /platform-redirect/?next=/enriching-events/tags/

Already tracked here - https://github.com/getsentry/sentry-docs/issues/3172